samples/unit-tests/gantt/gantt/demo.js

```diff
@@ -411,6 +411,16 @@
             2,
             'Collapsed graphics should not be rendered (#12617)'
         );
+
+        // Changing id of a parent
+        points[1].update({ id: '_moving' });
+
+        assert.strictEqual(
+            chart.series[0].yAxis.treeGrid.tree.children.length,
+            5,
+            'Orphaned nodes should appear as direct children of root'
```

Comment:
It would be good to add GH issue number if someone would like to get more information about this test.

Reply:
True! Also I am not completely sure this one unit-test "covers it". 

I suppose visual tests should pick up on this stuff since the stuff in this fix deals with visual errors, but still idk what the praxis is here 😬